sites/public/src/pages/listing/[id]/[slug].tsx

```diff
@@ -126,20 +127,25 @@ export async function getServerSideProps(context: {
   locale: string
 }) {
   let response
+  let jurisdiction: Jurisdiction
 
   const listingServiceUrl = runtimeConfig.getListingServiceUrl()
 
   try {
     response = await axios.get(`${listingServiceUrl}/${context.params.id}`, {
       headers: { language: context.locale },
     })
+    jurisdiction = await fetchJurisdictionByName(
+      runtimeConfig.getBackendApiBase(),
+      response.data.jurisdiction.name
+    )
```

Comment:
since we have the jurisdiction id we don't need to call the fetchJurisdictionByName, but instead just get it by id. https://github.com/metrotranscom/doorway/blob/main/backend/core/src/jurisdictions/jurisdictions.controller.ts#L46.

Or as an alternative to having to make an api call on every page load, could we just update the backend of the call made on line 133 to include the jurisdiction fields we need. This could be done by updating `jurisdiction.dto.ts` to include the fields we need such as `enableUtilitiesIncluded`